<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/Resource.qrc">
    <normaloff>:/icon/App</normaloff>:/icon/App</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionLoad_L"/>
    <addaction name="actionSaveAs_S"/>
    <addaction name="separator"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionNew_N"/>
    <addaction name="actionRemove_R"/>
    <addaction name="actionModify_M"/>
    <addaction name="actionRefresh_F"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
    <addaction name="actionUpdate_U"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="actionSink_S"/>
    <addaction name="actionOption_O"/>
   </widget>
   <widget class="QMenu" name="menuSticky_S">
    <property name="title">
     <string>Sticky(&amp;S)</string>
    </property>
    <addaction name="actionSticky_list_L"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuTools"/>
   <addaction name="menuSticky_S"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_L"/>
   <addaction name="actionSaveAs_S"/>
   <addaction name="separator"/>
   <addaction name="actionNew_N"/>
   <addaction name="actionRemove_R"/>
   <addaction name="actionModify_M"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh_F"/>
   <addaction name="actionSticky_list_L"/>
   <addaction name="actionExit_E"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionLoad_L">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/File</normaloff>:/icon/File</iconset>
   </property>
   <property name="text">
    <string>Load(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>Load tasks</string>
   </property>
   <property name="statusTip">
    <string>Load tasks</string>
   </property>
   <property name="whatsThis">
    <string>Load tasks</string>
   </property>
  </action>
  <action name="actionSaveAs_S">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Save</normaloff>:/icon/Save</iconset>
   </property>
   <property name="text">
    <string>Save as(&amp;S) ......</string>
   </property>
   <property name="iconText">
    <string>Save tasks as ......</string>
   </property>
   <property name="toolTip">
    <string>Save tasks as ......</string>
   </property>
   <property name="statusTip">
    <string>Save tasks as ......</string>
   </property>
   <property name="whatsThis">
    <string>Save tasks as ......</string>
   </property>
  </action>
  <action name="actionNew_N">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Add</normaloff>:/icon/Add</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>New tasks</string>
   </property>
   <property name="statusTip">
    <string>New tasks</string>
   </property>
   <property name="whatsThis">
    <string>New tasks</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionRemove_R">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Delete</normaloff>:/icon/Delete</iconset>
   </property>
   <property name="text">
    <string>Remove(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>Remove tasks</string>
   </property>
   <property name="statusTip">
    <string>Remove tasks</string>
   </property>
   <property name="whatsThis">
    <string>Remove tasks</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/App</normaloff>:/icon/App</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Close</normaloff>:/icon/Close</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
  <action name="actionModify_M">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Edit</normaloff>:/icon/Edit</iconset>
   </property>
   <property name="text">
    <string>Modify(&amp;M)</string>
   </property>
   <property name="toolTip">
    <string>Modify tasks</string>
   </property>
   <property name="statusTip">
    <string>Modify tasks</string>
   </property>
   <property name="whatsThis">
    <string>Modify tasks</string>
   </property>
  </action>
  <action name="actionRefresh_F">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Refresh</normaloff>:/icon/Refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Refresh</string>
   </property>
   <property name="whatsThis">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionOption_O">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Option</normaloff>:/icon/Option</iconset>
   </property>
   <property name="text">
    <string>Option(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>Option</string>
   </property>
   <property name="statusTip">
    <string>Option</string>
   </property>
   <property name="whatsThis">
    <string>Option</string>
   </property>
  </action>
  <action name="actionSink_S">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Sink</normaloff>:/icon/Sink</iconset>
   </property>
   <property name="text">
    <string>Sink(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>Sink</string>
   </property>
   <property name="toolTip">
    <string>Sink</string>
   </property>
   <property name="statusTip">
    <string>Sink</string>
   </property>
   <property name="whatsThis">
    <string>Sink</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Download</normaloff>:/icon/Download</iconset>
   </property>
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
  </action>
  <action name="actionSticky_list_L">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Sticky</normaloff>:/icon/Sticky</iconset>
   </property>
   <property name="text">
    <string>Sticky list(&amp;L)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
